src/data/global.ts

```diff
@@ -44,6 +45,7 @@ export enum PaymentMethod {
   klarnapaynow = 'klarnapaynow',
   klarnasliceit = 'klarnasliceit',
```

Comment:
Not necessarily for this PR, but some of these payment methods aren't documented (anymore) in the official `list-methods` documentation: https://docs.mollie.com/reference/list-methods

From what I see, the following are now deprecated:
- klarnapaylater
- klarnapaynow
- klarnasliceit

We should probably move them into `HistoricPaymentMethod` and ideally add a jsdoc to explain what `HistoricPaymentMethod` signifies - i.e. deprecated methods.